server/middleware/validation.js

```diff
@@ -29,6 +32,24 @@ const validators = {
         }
         return null;
     },
+    isValidDay({ label, value, message }) {
+        if (getDay(value) > new Date(getYear(value), getMonth(value), 0).getDate() || getDay(value) < 1) {
+            return message || validationErrors.isValidDay(label);
+        }
+        return null;
+    },
+    isValidMonth({ label, value, message }) {
+        if (getMonth(value) < 1 || getMonth(value) > 12) {
+            return message || validationErrors.isValidMonth(label);
+        }
+        return null;
+    },
+    isValidYear({ label, value, message }) {
+        if (getYear(value) > (new Date().getFullYear() + 100) || getYear(value) < (new Date().getFullYear() - 100)) {
```

Comment:
Consider splitting this line over several lines. It's too long.

Reply:
Updated